src/java/org/apache/cassandra/cql3/statements/schema/AlterTableStatement.java

```diff
@@ -804,6 +891,19 @@ public void dropCompactStorage()
             kind = Kind.DROP_COMPACT_STORAGE;
         }
 
+        public void dropConstraint(ColumnIdentifier name)
+        {
+            kind = Kind.DROP_CONSTRAINT;
+            this.constraintName = name;
```

Comment:
you dont need "this" here. To keep "this", either rename "constraintName" to "name" or rename the parameter from "name" to "constraintName" Same bellow in "addConstraint" method.